chainer/functions/math/clip.py

```diff
@@ -11,7 +11,7 @@ class Clip(function_node.FunctionNode):
 
     def __init__(self, x_min, x_max):
```

Comment:
Could you also check that not both of min and max are `None`, and if so, throw an appropriate error.